smithy-build/src/main/java/software/amazon/smithy/build/plugins/ModelPlugin.java

```diff
@@ -34,10 +36,16 @@ public String getName() {
 
     @Override
     public void execute(PluginContext context) {
-        context.getFileManifest().writeJson("model.json", serializeModel(context.getModel()));
+        boolean includePrelude = false;
+        Optional<BooleanNode> node = context.getSettings().getBooleanMember("includePreludeShapes");
```

Comment:
This seems off since you could set this to `false`. I'd use:

```
boolean includePrelude = context.getSettings().getBooleanMemberOrDefault("includePreludeShapes");
```